src/components/ui/Input.jsx

```diff
@@ -27,7 +28,7 @@ const Input = (props) => {
         <>
             {label && (
                 <label
-                    className={`mt-1 mb-2 whitespace-wrap text-sm md:text-base lg:text-lg flex:me-10 text-${labelColor} font-light flex:self-center capitalize text-sm md:text-base lg:text-lg`}
+                    className={`w-4/12 mt-1 mb-2 whitespace-wrap text-sm md:text-base lg:text-lg flex:me-10 text-${labelColor} font-light flex:self-center capitalize text-sm md:text-base lg:text-lg`}
```

Comment:
You should't change width of this.
It might cause change somewhere else that uses this component